tests/integrationtest/t/executor/delete.test

```diff
@@ -135,6 +135,7 @@ create table parent2 (a int primary key);
 create table child2 (a int, foreign key (a) references parent2(a));
 insert into parent2 values (1), (2);
 insert into child2 values (1);
+-- replace_regex /`executor__delete`\.`child2`, CONSTRAINT `fk_1` FOREIGN KEY \(`a`\) REFERENCES `parent2` \(`a`\)/`executor__delete`.`child`, CONSTRAINT `fk_1` FOREIGN KEY (`a`) REFERENCES `parent` (`a`)/
 -- error 1451
```

Comment:
Could use `--error 1451, 1451` here, it will only check error number and print `Got one of the listed errors` in result file.

Reply:
Done.